### ISSUE TITLE
Legal Notice don't popup correctly

### ISSUE THREAD

**Issue by MathieuDebit** [MathieuDebit/rise2016-website#1]
### The Problem

- The Legal Notice don't popup correctly after the `middleman build`. 
- No problem in development mode (within Middleman LiveReload).
- Only the `fr` build is uploaded

**Correct display** (Middleman LiveReload):
![rise2016-popup_ok](https://cloud.githubusercontent.com/assets/6206650/13114413/a211e084-d593-11e5-9d79-437f2af78aa9.png)

**Bug** (Middleman Build):
![rise2016-popup_ko](https://cloud.githubusercontent.com/assets/6206650/13114442/be127938-d593-11e5-9d75-7004aec9aa84.png)

## Fixes tried
- [x] Build without translation (`/locales`)
- [x] Remove duplicate jQuery scripts

**Comment by MathieuDebit** [comment]
The popup seems to inject the content of the index...

Result when I upload the entire build, and open legal notice at `rise2016.eu/fr`:
![rise2016-popup_index](https://cloud.githubusercontent.com/assets/6206650/13115078/adf25cd2-d596-11e5-8764-ef2af8f84228.png)

**Comment by MathieuDebit** [comment]
The modal (`SLIM`):

```slim
#modal_mention.modal.fade aria-labelledby="modal_mentionLabel" role="dialog" tabindex="-1"
  .modal-dialog role="document"
    .modal-content
      .modal-header
        button.close aria-label="Close" data-dismiss="modal" type="button"
          span aria-hidden="true"  &times;
        h4#modal_mentionLabel.modal-title #{t('footer.button')}
      .modal-body #{t('footer.modal_content')}
```

The modal (`HTML`):

```html
<div aria-labelledby="modal_mentionLabel" class="modal fade" id="modal_mention" role="dialog" tabindex="-1">
  <div class="modal-dialog" role="document">
    <div class="modal-content">
      <div class="modal-header">
        <button aria-label="Close" class="close" data-dismiss="modal" type="button">
          <span aria-hidden="true"> ×</span>
        </button>
        
        <h4 class="modal-title" id="modal_mentionLabel">translation missing: en.footer.button</h4>
      </div>
            
      <div class="modal-body">{content}</div>

    </div>
  </div>
</div>
```

But when we click to open the modal... tadah !

```html
<div aria-labelledby="modal_mentionLabel" class="modal fade in" id="modal_mention" role="dialog" tabindex="-1" style="display: block;">
  <div class="modal-dialog" role="document">
    <div class="modal-content">
      <title>Index of /</title>
      <h1>Index of /</h1>
      <pre>
        <img src="/__ovh_icons/blank.gif" alt="Icon "> 
        <a href="?C=N;O=D">Name</a>                    
        <a href="?C=M;O=A">Last modified</a>      
        <a href="?C=S;O=A">Size</a>  
        <a href="?C=D;O=A">Description</a>
        <hr>
        <img src="/__ovh_icons/layout.gif" alt="[   ]"> <a href="RISE2016_Call_for_Chairs.pdf">RISE2016_Call_for_Ch..&gt;</a> 22-Jan-2016 18:44  219K
        <img src="/__ovh_icons/unknown.gif" alt="[   ]"> <a href="contact-form.php">contact-form.php</a>        17-Feb-2016 16:44  680
        <img src="/__ovh_icons/folder.gif" alt="[DIR]"> <a href="de/">de/</a>                     17-Feb-2016 17:19    -
        <img src="/__ovh_icons/folder.gif" alt="[DIR]"> <a href="en/">en/</a>                     17-Feb-2016 17:19    -
        <img src="/__ovh_icons/image2.gif" alt="[IMG]"> <a href="favicon.ico">favicon.ico</a>             28-Aug-2015 11:09   15K
        <img src="/__ovh_icons/folder.gif" alt="[DIR]"> <a href="font/">font/</a>                   17-Feb-2016 16:44    -
        <img src="/__ovh_icons/folder.gif" alt="[DIR]"> <a href="fonts/">fonts/</a>                  17-Feb-2016 16:44    -
        <img src="/__ovh_icons/folder.gif" alt="[DIR]"> <a href="fr/">fr/</a>                     17-Feb-2016 17:19    -
        <img src="/__ovh_icons/folder.gif" alt="[DIR]"> <a href="images/">images/</a>                 17-Feb-2016 16:44    -
        <img src="/__ovh_icons/folder.gif" alt="[DIR]"> <a href="javascripts/">javascripts/</a>            17-Feb-2016 16:44    -
        <img src="/__ovh_icons/folder.gif" alt="[DIR]"> <a href="stylesheets/">stylesheets/</a>            17-Feb-2016 16:44    -
        <img src="/__ovh_icons/folder.gif" alt="[DIR]"> <a href="v1/">v1/</a>                     16-Feb-2016 16:44    -
        <img src="/__ovh_icons/folder.gif" alt="[DIR]"> <a href="v2/">v2/</a>                     08-Feb-2016 12:43    -
        <img src="/__ovh_icons/folder.gif" alt="[DIR]"> <a href="wordpress/">wordpress/</a>              10-Oct-2015 13:15    -
        <img src="/__ovh_icons/text.gif" alt="[TXT]"> <a href="zindex.html">zindex.html</a>             17-Feb-2016 19:08   14K
        <hr>
      </pre>
    </div>
  </div>
</div>
```

This content is dynamically injected, probably in JS.

**Comment by MathieuDebit** [comment]
Ok so the problem doesn't appear on build:

When clicking on the button to open the modal, a `XHR` request is made on index (`/`). 
![rise2016-modal_xhr](https://cloud.githubusercontent.com/assets/6206650/13172205/47c024f0-d6f7-11e5-8dd1-aa00cbfba268.png)

On `localhost`, index returns a `404` error, so nothing is injected into the modal (giving the ilusion that everything works), but on production, `index.php` redirects to something.

And the culprit code is this `href` on `_footer.slim`:

```slim
p: a href="/" data-target="#modal_mention" data-toggle="modal" type="button" #{t('footer.button')}
```

---
**Fixed on commit [92d1f5d](https://github.com/MathieuDebit/rise2016-website/commit/92d1f5d0a26f70e812b93d19b1da4709e6e0bcd6)**.
